.github/workflows/release_dev.yml

```
@@ -63,28 +60,32 @@ jobs:
+          TAG_NAME="$BRANCH_SUFFIX/$DATE_TAG"
+          echo "tag_name=$TAG_NAME" >> $GITHUB_OUTPUT
 
-      - id: delete_release_tag
-        name: Delete dev tag
-        if: steps.check_dev_tag.outputs.exists == 'true'
+      - name: Cleanup old releases
         run: |
-          gh release delete "dev/${{ github.ref_name }}" -y --cleanup-tag
+          BRANCH_SUFFIX="${{ steps.tag_info.outputs.tag_name%%/* }}"
+          PREFIX="${BRANCH_SUFFIX}/"
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)